src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public void validate(String password, String repeatPassword) {
-        //write your code here
+    private static final int MAX_PASS_LENGTH = 10;
+
+    public void validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null
+                || (password.length() < MAX_PASS_LENGTH
+                && repeatPassword.length() < MAX_PASS_LENGTH)
```

Comment:
looks like a separate check, but it's not. Please add some indent or move to the previous line